src/__tests__/03.extra-1.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import {render, screen} from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import App from '../final/03.extra-1'
+// import App from '../exercise/03'
+
+test('App works', () => {
+  render(<App />)
+  userEvent.type(screen.getByRole('textbox', {name: /name/i}), 'mulan')
+  userEvent.type(screen.getByRole('textbox', {name: /animal/i}), 'dragon')
+  screen.getByText('Your favorite animal is: dragon!')
```

Comment:
This fails the test when it should be but looking at other unit tests in the course, I debated if the following way could be better. Please let me what you think @kentcdodds.

```suggestion
  expect(screen.getByText('Your favorite animal is: dragon!')).toBeInTheDocument()
```